src/utils.d.ts

```diff
@@ -0,0 +1,8 @@
+export const INVALID_PATH_ERROR =
+    'node-jq: invalid path argument supplied (not a valid path)'
+export const INVALID_JSON_PATH_ERROR =
+    'node-jq: invalid path argument supplied (not a .json file)'
+
+export function isJSONPath(path: any): boolean
+
+export function validateJSONPath(path: any): boolean
```

Comment:
I saw the other validation returning the value "validated" and looks good to me, but this one, returns a boolean?

Reply:
isJSONPath returns the boolean result from it's regex test. validateJSONPath was updated to return true at the end if it hadn't thrown an error (previously was a void function)